app/networkapi/templates/highlights/adminsortable_objects_custom.html

```diff
@@ -0,0 +1,19 @@
+{% load adminsortable_tags admin_urls staticfiles %}
```

Comment:
Turn the "this file" specific code into a `{% block %}`, and put the rest in a separte file, then have this file `{% extends ... %}`, and that should do the trick.